### PR TITLE
Pope of Sin Changes/Fixes

### PR DESCRIPTION
**Pull request by sawu-tg**
- Added a new, custom effect to Hades' Sin Shed.
- Hades will no longer repeatedly teleport to the center of the chapel during Flee Phase.
- Hades now slowly gains HP, up to half, when on hallowed ground.
- Spawned and Teleported Acolytes no longer spawn on top of Hades.
- Hades will now lasso targets who try to run.
- Custom beam sprite added for Hades' beams.
- Hades' Rage Ultimate no longer ignores simple animals and non-client objects.
- Hades has a new Ultimate, which will knock down everyone nearby.
- Various pieces of flavor/fluff text added to sins.
- Indulged Sloth no longer gives 1000 stacks of drowyness, now gives 100.
- Hades' Rage Ultimate now has a more appropriately sized effect.
- Hammer of Pride damage balanced out more.
- Hammer of Pride now transfers health between the target and the user, depending on it's indulged status.
- Acolytes of Hades can now be butchered for their outfits and non-functioning versions of their staves.
- Wizards can now buy a Dark Seed, a relic that feeds on living or dead flesh to summon Hades.
- Staff of Hades has been rebalanced through the different versions.
- Hades' sprite updated slightly.

:cl: Wizard's Federation
add: Acolytes of Hades have been scolded for gluing their clothes to themselves, and their clothes can now be cut off them.
add: Hades has learned a few new tricks, to ensure the ultimate demise of his target.
fix: Hades has faced a council of his repeated use of time magic inside chapels, and as it was deemed unholy, he will no longer do it.
add: Wizards of the Wizard's Federation have been granted access to Dark Seeds, ancient relics used to call upon Hades in a time of need.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
What's he look like now

**Comment by sawu-tg** [comment]
@Zilenan91  I just updated his mask to use the same sprite as the acolyte's masks. It's an extremely minor change, mostly for consistency's sake.

**Comment by KorPhaeron** [comment]
Only costs 1?

Also I am not sure making the mob inescapable is good if the average crewman cant realistically fight it.

**Comment by Zilenan91** [comment]
Yeah it probably shouldn't cost 1 for what it does with how easy it is for wizards to rack up a body count. 2 should be good.

**Comment by sawu-tg** [comment]
@KorPhaeron It's mainly to prevent people leading him away, which seems to be a common tactic. I could increase the grab range to something more like 4 or 5 tiles and add a timer, rather than a flat.

Also @Zilenan91 I didn't really know what the price averages were, so i just threw it at whatever the other summons were. I'll up it now.

**Comment by KorPhaeron** [comment]
We double spellbook total points (and costs) recently, so 2 is the cost of one normal spell/summon

File: code/game/gamemodes/miniantags/hades/hades.dm [hunk 1]
```diff
@@ -206,9 +209,9 @@
 	var/obj/effect/overlay/temp/cult/sparks/S = PoolOrNew(/obj/effect/overlay/temp/cult/sparks, T)
 	S.anchored = FALSE
 	S.throw_at_fast(src,10,1)
-	var/obj/effect/overlay/temp/bloodsplatter/BS = PoolOrNew(/obj/effect/overlay/temp/bloodsplatter, list(T, get_dir(src, T)))
-	BS.anchored = FALSE
-	BS.throw_at_fast(src,10,1)
+	S.density = 0
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/hades/hades.dm]: these aren't dense!!

File: code/game/gamemodes/miniantags/hades/hades.dm [hunk 2]
```diff
@@ -206,9 +209,9 @@
 	var/obj/effect/overlay/temp/cult/sparks/S = PoolOrNew(/obj/effect/overlay/temp/cult/sparks, T)
 	S.anchored = FALSE
 	S.throw_at_fast(src,10,1)
-	var/obj/effect/overlay/temp/bloodsplatter/BS = PoolOrNew(/obj/effect/overlay/temp/bloodsplatter, list(T, get_dir(src, T)))
-	BS.anchored = FALSE
-	BS.throw_at_fast(src,10,1)
+	S.density = 0
+	var/obj/effect/overlay/temp/hadesBlood/BS = PoolOrNew(/obj/effect/overlay/temp/hadesBlood, T)
+	BS.density = 0
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/hades/hades.dm]: this does nothing!

**Pull request merged**